src/entity/user/utils.ts

```diff
@@ -256,6 +258,12 @@ const safeInsertUser = async (
       const newUserId = newUser.identifiers[0].id;
       const feedId = newUserId;
 
+      await insertOrIgnoreAction(
+        entityManager,
+        newUserId,
+        UserActionType.CheckedCoresRole,
+      );
```

Comment:
User's who register with region already have it resolved so we don't fulfil user action immediately.